lib/helpers/backend/googleBooksSearch.ts

```diff
@@ -0,0 +1,25 @@
+// Note that I was having trouble with https://www.npmjs.com/package/google-books-search
+// The following code provides the same data and provides more information
+
+import { GoogleBook } from "../../../common/types";
+import { google, books_v1 } from "googleapis";
+import { GaxiosResponse } from "gaxios";
```

Comment:
gaxios appears to be a transient dependency pulled in from the google books API package. Maybe add gaxios directly into the package.json?

Reply:
Sounds good. Updated in latest commit. Thanks David!